user-guide/Advanced_Modules/scheduler/Scheduler_NuGet_Package.md

```diff
@@ -0,0 +1,217 @@
+  var monthsToRepeat = new List<Months> { Months.February, Months.August };
+  var daysInMonthToRepeat = new List<int> { 1, 2, 15, 16 };
+  var scheduleMonthly = new ScheduleMonthly(now, monthsToRepeat, daysInMonthToRepeat, repeats, repetitionIntervalInMinutes, endDate);
+  ```
+  
+  With this example, a monthly task will run every 100 minutes for the next year, in February and August, on the 1st, 2nd, 15th and 16th days of the referred months, until it has been executed at most 400 times before the task ends.
+
+Once you have defined the recurrence, you can **create the scheduled task**:
+
+```csharp
+var startRunTime = DateTime.Now;
+var endRunTime = startRunTime.AddHours(2);
+
+var schedulerTask = new SchedulerTask("Name", "Description", startRunTime, endRunTime, scheduleRepetition);
+```
+
+Assuming that it is 1 PM in this example, the task will be scheduled to run from 1 PM until 3 PM, being repeated according to the selected schedule repetition.
+
+With the task already defined, it is time to configure the **schedule action**. Currently, this library only supports Automation script actions, which can be defined as follows:
+
+ ```csharp
+var inputScriptParams = new List<AutomationScriptInputParameter>
+{
+  new AutomationScriptInputParameter("InputScriptParam1", "InputValue1"),
+  new AutomationScriptInputParameter("InputScriptParam2", "InputValue2"),
+};
+
+var automationScriptAction = new AutomationScriptAction("Script Name", inputScriptParams, checkSets: true, runAsync: false);
+
+schedulerTask.ISchedulerAction = automationScriptAction;
+schedulerTask.ISchedulerFinalAction = automationScriptAction;
+```
+
+This action will cause an Automation script (*runAsync* flag) named *Script Name* to be executed asynchronously. It will receive the *InputScriptParam1* and *InputScriptParam2* arguments by verifying whether a read parameter will be checked for a new value after a set command, using the optional *checkSets* flag.
+
+Note that it is possible to define a **final action** that will be executed when the Scheduler task is removed. In the presented case, the same Automation script action will be executed.
+
+Finally, when the task is fully ready, it can be **created or updated at DataMiner Scheduler module level** using the following method:
+
+ ```csharp
+scheduler.CreateOrUpdateSchedulerTask(schedulerTask);
+```
+
+You can create or update **multiple tasks** at the same time, making it possible to define an optional flag to indicate whether to send changes in bulk:
+
+ ```csharp
+var schedulerTasks = new List<SchedulerTask>
+{
+  schedulerTask1,
+  schedulerTask2,
+  schedulerTask3,
+};
+
+scheduler.CreateOrUpdateSchedulerTasks(schedulerTasks, sendInBulk: true);
+```
+
+## API Reference
+
+### Scheduler Class
+
+Initializes a new instance of the `Scheduler` class by retrieving the Scheduler tasks according to the given selector.
+
+ ```csharp
+public Scheduler(Func<Net.Messages.SchedulerTask, bool> selector = null)` 
+```
+
```

Comment:
Could you revise this page from this point onwards, @FlavioJA? This subsection isn't structured very clearly, largely because there are several identical subtitles (Parameters, Exceptions) all under the same overarching title (Scheduler Class). These identical titles also make it impossible to cross-reference to these specific subtitles on the page.